contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -448,8 +453,8 @@ func (q *queuer) Run(done <-chan struct{}) {
 			} else {
 				// use ReplaceExisting because we are always pushing the latest state
 				now := time.Now()
-				pod.deadline = &now
-				if q.podQueue.Offer(pod, queue.ReplaceExisting) {
+				pod.eventTime = &now
```

Comment:
The default clock impl delegates to `time`, so it's not a big deal unless we want to use FakeClock to test the scheduler plugin itself. At that point the clock injection would need to be moved up a level.